bHY6yh-de2065/cod-yyyiou_bHY6yh-de2065.xml

```diff
@@ -0,0 +1,210 @@
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e144"> 
+      <quote xml:id="bHY6yh-de2065-Qd1e146">Virum 
+               sanguinum et dolosum abominabitur dominus. </quote> 
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e152"> 
+       <ref xml:id="bHY6yh-de2065-Rd1e151">@Idem XXXIII</ref> <quote xml:id="bHY6yh-de2065-Qd1e154">Oculi 
+        <lb ed="#0"/>domini super iustos: et aures eius ad 
+        <lb ed="#0"/>praeces eorum. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e162"> 
+       <ref xml:id="bHY6yh-de2065-Rd1e164">@Idem 83 </ref> <quote xml:id="bHY6yh-de2065-Qd1e164">misericordiam et uer
+       <lb ed="#0"/>tatem diligit deus </quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e170">
+       <ref xml:id="bHY6yh-de2065-Rd1e174">@Idem capitulo LXVII</ref> <quote xml:id="bHY6yh-de2065-Qd1e172">
+        <lb ed="#0"/>dominus diligit iustos. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e176">
+                <ref xml:id="bHY6yh-de2065-Rd1e184">@Salomon Io Co </ref> <quote xml:id="bHY6yh-de2065-Qd1e180">Sex
+                <lb ed="#0"/> sunt que odit deus et septimum detes
+                <lb ed="#0"/>tatur
+                 anima eius Oculos sublimes Linguam 
+                 <lb ed="#0"/>mendacem manum effundentem inno
+                 <lb ed="#0"/>xium
+                  sanguinem Cor machinans cogita
+                  <lb ed="#0"/>tiones
+                   peximas pedes ueloces ad cor
+                   <lb ed="#0"/>rendum 
+                   ad malum proferentem mendacia 
+                   <lb ed="#0"/>Testem fallacem et eum qui inter fratres 
+                   <pb ed="#0" n="4-r"/>
+                   <cb ed="#0" n="a"/>
+                   <lb ed="#0"/>semmat discordiam. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e197">
+               / <ref xml:id="bHY6yh-de2065-Rd1e207">@Idem capitulo XV </ref> <quote xml:id="bHY6yh-de2065-Qd1e201">Qui 
+               <lb ed="#0"/>sequitur iustitiam diligitur a domino </quote>
+            </p> <!--not quite the same the difference is between the last words ab eo vs a domino-->
+            <p xml:id="bHY6yh-de2065-d1e206">
+               <quote xml:id="bHY6yh-de2065-Qd1e210"><lb ed="#0"/> Abhommatio domini cogitationes ma
+               <lb ed="#0"/>le
+               et puius subino et pulchiunes sinna
+               <lb ed="#0"/>bitur ab eo 
+                </quote>
+            </p> <!--some spelling differences-->
+            <p xml:id="bHY6yh-de2065-d1e214">
+      <quote xml:id="bHY6yh-de2065-Qd1e216">Longe est dominus ab im
+      <lb ed="#0"/>piis et orationes iustorum exaudiet.</quote>
+            </p>
+            <p xml:id="bHY6yh-de2065-d1e224">
+       <ref xml:id="bHY6yh-de2065-Rd1e234"><lb ed="#0"/>@Idem capitulo XVII</ref> <quote xml:id="bHY6yh-de2065-Qd1e226"> qui iustificat impium 
+        <lb ed="#0"/>et qui condempnat iustum abomina
```

Comment:
Is the "condempnat" a typo here? If not it should have <sic> around it to indicate an error.

Reply:
I think she means `<sic>`